### DISCUSSION TITLE
怎么样使文件夹和文件名字分行显示

### DISCUSSION THREAD

**Discussion by qgdyyg** [AlistGo/alist#6280] [General]
![屏幕截图 2024-03-28 214244](https://github.com/alist-org/alist/assets/78137202/ab7b80f1-e175-4e4f-abaa-1cde2c238b41)
求助，像这种情况，想让文件名字全部展示，有什么好的方法吗，谢谢了

**Comment by pongfcnkl** [comment]
![截图_20240328234545](https://github.com/alist-org/alist/assets/24273319/fdf6d8af-2e5f-4a80-8c73-0d49aca0da61)
